### ISSUE TITLE
Regarding new loss functions

### ISSUE THREAD

**Issue by radonys** [deepinsight/insightface#236]
Are the COCO loss, Marginal loss and Large Margin softmax loss functions complete in train.py? Plus center loss would be completed in near future?

**Comment by radonys** [comment]
@nttstar It would be helpful if you can tell about the status of the above loss functions.

**Comment by nttstar** [comment]
Yes. But not fully tested.
